packages/aws-cdk/lib/init.ts

```diff
@@ -292,26 +293,34 @@ async function postInstallTypescript(canUseNetwork: boolean, cwd: string) {
   const command = 'npm';
 
   if (!canUseNetwork) {
-    print(`Please run ${colors.green(`${command} install`)}!`);
+    warning(`Please run ${colors.green(`${command} install`)}!`);
```

Comment:
Drop the green part of this command and make it all yellow.

```suggestion
    warning(`Please run ${command} install!`);
```

Reply:
good call